### PR TITLE
Fix .vscode in npm

### PR DESCRIPTION
**Pull request by NV4RE**
There is .vscode file in npm registry (node-rdkafka@2.7.2), and it size are greater than 900MB. 

```bash
# du -d 1 -h
 24K	./bench
 88K	./test
 16K	./util
 16K	./ci
504K	./node_modules
 28K	./examples
120K	./lib
 15M	./deps
9.6M	./build
879M	./.vscode
 56K	./e2e
228K	./src
905M	.
```

### CODE REVIEW

**Comment by IdanAdar** [comment]
@NV4RE can you resolve the conflict?
@iradul can this then be merged at least if not yet published?

**Comment by NV4RE** [comment]
@IdanAdar @iradul Fixed conflict.

**Comment by iradul** [comment]
So `.vscode` gets included when `npm publish` is done manually. Now, package gets published with Travis so `.vscode` doesn't exist over there and all looks good, but I would still like to merge `.npmignore` fix just in case.
Please remove `index.d.ts` update since it doesn't belong in this PR and I'll merge this.

**Comment by NV4RE** [comment]
@iradul 
Removed change on index.d.ts

File: index.d.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ export class KafkaConsumer extends Client {
 
     setDefaultConsumeTimeout(timeoutMs: any): void;
 
-    subscribe(topics: string[]): this;
+    subscribe(topics: (string)[]): this;
```
Comment (iradul) [inline index.d.ts]: `index.d.ts` change still exists. Please make sure only `.npmignore` is listed here.

**Pull request merged**